game/src/main/scala/io/aigar/game/Game.scala

```diff
@@ -15,16 +15,16 @@ class Game(val id: Int) {
 
   def state = {
     //TODO really implement
-    GameState(
+    io.aigar.game.serializable.GameState(
```

Comment:
Well we'll be using `Cell`, `Player`, etc. classes elsewhere, so they would clash because we need both imports at some point. I improved it by just specifying `serializable` now. Ok with you? This is really a temporary impl anyway (my next PR will make it so that we return the real state).